docs/modules/ROOT/pages/index.adoc

```diff
@@ -77,3 +77,84 @@ quarkus.minio.secret-key=DUMMY-SECRET-KEY
 quarkus.minio.region=beyond
 quarkus.minio.allow-empty=true/false (Boolean type, default value is false)
 ----
+
+[[dev-services]]
+== Dev Services
+
+Quarkus supports a feature called Dev Services that allows you to create various containers without any config.
+What that means in practice is that if you have Docker running and have not configured `quarkus.minio.url`,
+Quarkus will automatically start a Minio container when running tests or in dev mode, and automatically
+configure the connection.
+
+When running the production version of the application, the Minio connection needs to be configured as normal.
+
+=== Shared server
+
+Most of the time you need to share the server between applications.
+Dev Services for Minio implements a _service discovery_ mechanism for your multiple Quarkus applications running in _dev_ mode to share a single server.
+
+NOTE: Dev Services for Minio starts the container with the `quarkus-dev-service-minio` label which is used to identify the container.
```

Comment:
Actually you are starting the container with the `quarkus.minio.devservices.service-name` label...

Reply:
![image](https://user-images.githubusercontent.com/8418431/195972964-f2622440-687a-4085-b493-cad9ff89f7a7.png)